### ISSUE TITLE
Update the trees that are being stitched onto the backbone.

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/tree-build#104]
I'm thinking particularly of the mammal tree which is very old and a user just kindly pointed out that it's not well supported any more (and diverges from open tree)

**Comment by hyanwong** [comment]
Yes, we might do better just switching to the OpenTree. Does OpenTree 15.1 have a decent mammal tree now? I had a look through primates and it seems pretty good, although of course the dates are missing.

**Comment by hyanwong** [comment]
OpenTree lists 9,324 mammals (admittedly including fossils and subspecies), whereas we only have 5,046. OTOH, there are still some pretty poorly resolved (taxonomy-only) areas of the tree. I'm surprised that they haven't got a more representative mammal tree in there yet. Here are some marsupials, for example:

<img width="804" alt="Screenshot 2024-11-19 at 21 59 53" src="https://github.com/user-attachments/assets/1f0b67ed-efea-4ac9-b88e-a22b8ad645cb">

I think the problem is that the [Upham/Jetz tree](https://pmc.ncbi.nlm.nih.gov/articles/PMC6892540/), with 5,911 species of mammals is not in there. I see that Emily Jane put it in the file list (at https://tree.opentreeoflife.org/curator/study/view/ot_1990/?tab=home), but there is no actual tree data. I think we should probably get this tree included before switching wholesale to the OpenTree for mammals? Can you ask Emily Jane what the issue is here (I see that she added a comment to https://github.com/McTavishLab/OT2020/blob/master/VertLifeMammalTree.ipynb, but that's now a dead link). Maybe we'll have some dates by the time she manages to resolve this too?

**Comment by hyanwong** [comment]
Actually, maybe we should just use one of the [Upham/Jetz trees](https://pmc.ncbi.nlm.nih.gov/articles/PMC6892540/), since these have dates, and there is a OneZoom version on their site anyway: http://vertlife.org/data/mammals/. @jrosindell : you are best placed to think about how we incorporate the http://vertlife.org trees into OneZoom?

**Comment by davidebbo** [comment]
The working link for VertLifeMammalTree.ipynb is https://github.com/McTavishLab/OT2020/blob/master/mammal/VertLifeMammalTree.ipynb.

> Actually, maybe we should just use one of the [Upham/Jetz trees](https://pmc.ncbi.nlm.nih.gov/articles/PMC6892540/), since these have dates, and there is a OneZoom version on their site anyway: http://vertlife.org/data/mammals/.

Yes, that seems like a good thing to explore. They have 5911 mammals to our 5046. No subspecies, but they do have a number of extinct species (although far from complete). That might explain some of the difference. e.g. looking at bears, they list just 3 extinct species (my tree has about 25).

**Comment by hyanwong** [comment]
> The working link for VertLifeMammalTree.ipynb is https://github.com/McTavishLab/OT2020/blob/master/mammal/VertLifeMammalTree.ipynb.

Thanks. I just corrected it on the study curator.

Weird that there's no tree there though.

**Comment by davidebbo** [comment]
> Weird that there's no tree there though.

You can get the tree from https://itol.embl.de/tree/10823010783366411588275209. Export / Newick. Though I don't really understand how they have the OTTs in there, e.g.

((Panthera_uncia,Panthera_tigris)mrcaott19397ott532117,(Panthera_pardus,(Panthera_onca,Panthera_leo)mrcaott371429ott735488)mrcaott42318ott371429)Panthera)

532117 is the OTT for Panthera_uncia, but the one for P tigris (42314) is nowhere to be found. It has 19397, which is a tiger subspecies. Plus they're all positioned weirdly. Does that make any sense to you?

Side note: nice to see the snow leopard under Panthera, unlike our tree!

**Comment by hyanwong** [comment]
> You can get the tree from https://itol.embl.de/tree/10823010783366411588275209. 

Yes, I means that it's weird there's no tree (that I can see) associated with the study in the OpenTree curator app. I'm not sure why Emily didn't add it.
